src/_locales/zh_TW/messages.json

```diff
@@ -409,40 +409,40 @@
     "hash": "21667c5a30f89561cea02351172b9c91"
   },
   "optionUseMonospaceFont": {
-    "message": "使用等寬字體",
+    "message": "使用等寬字型",
     "description": "This is an option shown in the add-on settings.",
     "hash": "69191273c436064be8c4f5d6a8f675f5"
   },
   "optionDebugMode": {
-    "message": "啟用除錯模式",
+    "message": "啟用偵錯模式",
     "description": "This is an option shown in the add-on settings.",
     "hash": "de6506af4973e0980d64f242fba11a19"
   },
   "optionDebugModeDescr": {
-    "message": "僅用在控制台取得更詳細的日誌輸出用於匯報問題...等等。",
+    "message": "僅用來從控制臺輸出更詳細的記錄，用於回報錯誤等。",
     "description": "The description shown for the debug mode (optionDebugMode).",
     "hash": "2aff10770796694827ec4253860e0771"
   },
   "translatorCredit": {
-    "message": "該擴充元件正體中文翻譯由 $TRANSLATORS$ 參考 Cye3s的 zh_CN 完成。",
```

Comment:
Yeah as written below I guess the old translator did a mistake here and did not link all the translators. `translatorUsername` does not have to be one username only.